src/util.js

```diff
@@ -300,7 +304,11 @@ define([
                 "XML " + (opts.not ? "should not be equivalent" : "mismatch"));
         return patch;
     };
-        
+
+    that.markdownFull = function (text) {
```

Comment:
was originally based on copy-paste-dev so this was to differentiate with your markdownlite. I suppose that function could be thrown away on copy-paste once these are merged together.